fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -353,7 +353,7 @@ def self.hide_transporter_output?
     #                            see: https://github.com/fastlane/fastlane/issues/1524#issuecomment-196370628
     #                            for more information about how to use the iTMSTransporter to list your provider
     #                            short names
-    def initialize(user = nil, password = nil, use_shell_script = false, provider_short_name = nil)
+    def initialize(user = nil, password = nil, use_shell_script = false, provider_short_name = nil, using_api_key = false)
```

Comment:
Overall, it looks pretty good. I am not a huge fan on adding another parameter to the initializer tho. Perhaps we can add a new class method, something like `ItunesTransporter.using_api_key(key: '', issuer: '')` that would call the original initializer and set the `using_api_key` attribute.

Reply:
Code is from @Ruenzuo, maybe they can say something about that?

Looking at the code myself, I think this is an easier option of adding this to the existing code. `user` and `pass` are repurposed, so nothing else really has to be changed in between.